src/project/internal/projectactionscontroller.cpp

```diff
@@ -307,6 +317,11 @@ bool ProjectActionsController::closeOpenedProject(bool quitApp)
     if (result) {
         globalContext()->setCurrentProject(nullptr);
 
+        Ret ret = closeNotationPageIfNeed();
```

Comment:
We should open the home page only if quitApp == false. So the implementation will be like this:

if (quitApp) {
dispatcher()->dispatch("quit", actions::ActionData::make_arg1<bool>(false));
} else {
Ret ret = closeNotationPageIfNeed();
...
}